docs/fxa-engineering/fxa-metrics.md

```diff
@@ -127,7 +127,7 @@ The tables are visible in Redash, under the “FxA Activity Metrics” data sour
 
 Our Amplitude events are sent to a GCP pubsub queue from stackdriver, this stuff is managed by jbuck. We have a node script that listens to the queue and then sends the events to Amplitude:
 
-* https://github.com/mozilla/fxa-amplitude-send/blob/master/pubsub/index.js
+* https://github.com/mozilla/fxa-amplitude-send/blob/main/pubsub/index.js
```

Comment:
This is still a 404.  Let's just link to the top level repository: https://github.com/mozilla/fxa-amplitude-send